orchagent/aclorch.cpp

```diff
@@ -2001,6 +2002,12 @@ bool AclRulePacket::validateAddAction(string attr_name, string _attr_value)
             actionData.parameter.booldata = true;
             action_str = ACTION_DO_NOT_NAT_ACTION;
         }
+        // handle PACKET_ACTION_DISABLE_TRIM in ACTION_PACKET_ACTION
+        else if (attr_value == PACKET_ACTION_DISABLE_TRIM)
+        {
+            actionData.parameter.booldata = true;
+            action_str = ACTION_DISABLE_TRIM;
+        }
```

Comment:
Could you add mock tests for ACL rule setting trim action

Reply:
@kperumalbfn done